target/linux/ramips/mt7621/base-files/etc/init.d/bootcount

```diff
@@ -15,6 +15,9 @@ boot() {
 			$((0xff)) ]] || printf '\xff' | dd of=/dev/mtdblock3 \
 			count=1 bs=1 seek=$((0x20001))
 		;;
+	jdcloud,re-cp-02)
+		echo -e "bootcount 0\nbootlimit 5\nupgrade_available 1"  | /usr/sbin/fw_setenv -s -
```

Comment:
Where can I find more information about why setting bootlimit is needed?
I don't know too much about it

Reply:
https://github.com/u-boot/u-boot/blob/master/doc/README.bootcount